Dockerfile

```diff
@@ -1,15 +1,25 @@
-FROM golang:1.19.8-alpine as builder
-RUN apk add gcc musl-dev linux-headers git bash curl
-RUN git clone https://github.com/AthanorLabs/atomic-swap
+FROM golang:latest as builder
+# Download monero-wallet-rpc. We need bzip2 to unpack the tar file.
+RUN apt update && apt install -y bzip2
+RUN arch=$(uname -m | sed 's/x86_64/linux64/; s/aarch64/linuxarm8/') && \
+    curl -sSL "https://downloads.getmonero.org/cli/${arch}" -o monero.tar.bz2
+RUN tar xvjf monero.tar.bz2 --no-anchored monero-wallet-rpc --strip-components=1
+
+# Build the swapd and swapcli binaries
+RUN git clone --depth=1 https://github.com/AthanorLabs/atomic-swap
 WORKDIR atomic-swap
-RUN bash scripts/install-monero-linux.sh
-RUN go build --tags=prod,netgo,osusergo --ldflags '-extldflags "-static"' ./cmd/swapd/
```

Comment:
Since we know the container has to have glibc installed inside it for `monero-wallet-rpc`, a static build will just produce a larger container without removing the glibc dependency. The easiest way to get a static build if we didn't depend on glibc for `monero-wallet-rpc` would be:
```
CGO_ENABLED=0 make build
```

Reply:
I don't think we can disable CGO since go-ethereum depends on it, which is why I had the static build flags

Reply:
We either don't use the features in `go-ethereum` that require CGO, or their code is configured to fallback to a go implementation of certain parts, as our binaries build okay:
```
rm -rf bin
CGO_ENABLED=0 make build
ldd bin/swapd
```
and ldd outputs:
```
not a dynamic executable
```

Reply:
good to know!